# /Bug0:
#   ros__parameters:
#     goal_x: 0.0
#     goal_y: -6.0
#     goal_theta: 0.0

/Bug0:
  ros__parameters:
    goal_x: 9.0
    goal_y: -2.0
    goal_theta: 0.0

# /Bug0:
#   ros__parameters:
#     goal_x: 0.0
#     goal_y: 4.0
#     goal_theta: 0.0
